roles/openshift_health_checker/action_plugins/openshift_health_check.py

```diff
@@ -196,17 +201,75 @@ def run_check(name, check, user_disabled_checks):
         is_active = check.is_active()
     except Exception as exc:
         reason = "Could not determine if check should be run, exception: {}".format(exc)
-        return dict(skipped=True, skipped_reason=reason, exception=traceback.format_exc())
+        return dict(skipped=True, skipped_reason=reason, exception=traceback.print_exc())
```

Comment:
I didn't understand the reason for this change. `format_exc` returns a string, `print_exc` writes to a file -- `sys.stderr` by default. IIUC, using `print_exc` here causes the side effect but makes an entry `exception: None` in the returned `dict`.

Reply:
that's my mistake, I was fiddling with something, got distracted and left it there. thanks!